aws-applicationinsights-application/aws-applicationinsights-application.json

```diff
@@ -346,7 +354,45 @@
                     "type": "string"
                 },
                 "PrometheusPort": {
-                    "description": "Prometheus exporter port",
+                    "description": "Prometheus exporter port.",
+                    "type": "string"
+                }
+            },
+            "additionalProperties": false
+        },
+        "HANAPrometheusExporter": {
+            "description": "The HANA DB Prometheus Exporter settings.",
+            "type": "object",
+            "properties": {
+                "HANASID": {
```

Comment:
We should add valid patterns for new added properties, which are in sync with the validation of our service APIs. Pre validation on CFN template level can help customers to capture invalid input early before stack creation/update (in comparison to during the stack creation/update, when actually calling AI APIs).